packages/fxa-content-server/app/scripts/lib/verification-reasons.ts

```diff
@@ -8,6 +8,7 @@
 
 enum VerificationReasons {
   FORCE_AUTH = 'force_auth',
+  CHANGE_PASSWORD = 'change_password',
```

Comment:
Feel free to change the auth-server value as you see fit: https://github.com/mozilla/fxa/blob/master/packages/fxa-auth-server/lib/routes/account.js#L839